### PR TITLE
Feature/work cards

### PR DESCRIPTION
**Pull request by ngocphong230801**
In this pull requets :
- Add code for index.
- Add CSS work cards.
- Update CSS heading.
- Add the image

### CODE REVIEW

File: practices/practice-one/src/styles/abstracts/variables.css [hunk 1]
```diff
@@ -19,7 +19,7 @@
     /* Heading-color*/
     --color-heading: #11223f;
     --color-heading-title: var(--primary-color);
-    
+    --color-heading-sm: #1967FA;
```
Comment (huyenpham-agilityio) [inline practices/practice-one/src/styles/abstracts/variables.css]: consistent color code please, choose one type: normal or capitalize

File: practices/practice-one/src/index.html [hunk 2]
```diff
@@ -29,14 +29,109 @@
...
             </div>
            <!-- End banner -->
+           <!-- Section feedback -->
+            <section>
+                <div class="section-feedback">
+                    <div class="card">
+                        <div class="card-user">
+                            <img src="../src/assets/img/ManPicture.png" alt="Userman" class="user-img">
+                            <div class="card-infor">
+                                <h3 class="heading-info">John Doe</h3>
```
Comment (huyenpham-agilityio) [inline practices/practice-one/src/index.html]: correct name please: infor or info?

Comment (huyenpham-agilityio) [inline practices/practice-one/src/index.html]: is it correct if use h3 for card name?

File: practices/practice-one/src/index.html [hunk 3]
```diff
@@ -29,14 +29,109 @@
...
+                        </div>
+                    </div>         
+                    <div class="card">
+                        <div class="card-user">
+                            <img src="../src/assets/img/WomanPicture.png" alt="UserWoman" class="user-img">
+                            <div class="card-infor">
+                                <a class="heading-infor">Jane Goodman<a>
+                                <p class="infor">Founder, ImpactNation</p>
+                            </div>
+                            <div class="quoet">
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html]: typo

File: practices/practice-one/src/index.html [hunk 4]
```diff
@@ -29,14 +29,109 @@
...
+                        <div class="card-content">
+                            <p class="feedback"> "Our senior fronted developer was 70% faster in building React.js componets thanks to VanGoux!"
+                            </p>
+                        </div>
+                    </div>         
+                    <div class="card">
+                        <div class="card-user">
+                            <img src="../src/assets/img/WomanPicture.png" alt="UserWoman" class="user-img">
+                            <div class="card-infor">
+                                <a class="heading-infor">Jane Goodman<a>
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html]: heading-infor -> profile-name

File: practices/practice-one/src/index.html [hunk 5]
```diff
@@ -29,14 +29,109 @@
...
+                                <p class="infor">Founder, ImpactNation</p>
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html]: infor -> profile-title

File: practices/practice-one/src/index.html [hunk 6]
```diff
@@ -29,14 +29,109 @@
...
+                        <div class="card-content">
+                            <p class="feedback">It was awesome to see VanGoux turn our MVP designs into code within minutes"
+                            </p>
+                        </div>
+                    </div>   
+                </div>    
+                <!-- End feedback -->
+                <!-- Work-cards -->
+                <div class="title">
+                    <h3 class="heading-title"> HOW VANGOUX WORKS</h3>
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/index.html]: Space, use CSS to uppercase

Comment (cunguyen-agilityio) [reply practices/practice-one/src/index.html]: Apply for all of place

File: practices/practice-one/src/styles/abstracts/variables.css [hunk 7]
```diff
@@ -19,7 +19,7 @@
     /* Heading-color*/
     --color-heading: #11223f;
     --color-heading-title: var(--primary-color);
-    
+    --color-heading-sm: #1967fa;
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/styles/abstracts/variables.css]: Please update

File: practices/practice-one/src/styles/layout/work-cards.css [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+    position: absolute;
+    left: 40px;
+    top: 60px;
+    font-weight: var(--font-weight-bold);
+    color: var(--primary-color);
+    font-size: var(--font-size-text-lg);
+    margin-top: 5px;
+}
+
+.work-cards .card .card-bottom .content {
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/styles/layout/work-cards.css]: For card styles, should move it as a component then styled for it

Comment (cunguyen-agilityio) [reply practices/practice-one/src/styles/layout/work-cards.css]: max of selectors is 3 level

File: practices/practice-one/src/styles/layout/work-cards.css [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    
+}
+
+.work-cards .card {
+    width: 318px;
+    height: 220px;
+    background-color: #ffffff;
```
Comment (cunguyen-agilityio) [inline practices/practice-one/src/styles/layout/work-cards.css]: color short-hand

**Review by cunguyen-agilityio — APPROVED** [review]

**Review by cunguyen-agilityio — APPROVED** [review]

**Pull request merged**